firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FeedbackActivity.java

```diff
@@ -30,10 +35,9 @@
 
 /** Activity for tester to compose and submit feedback. */
 public class FeedbackActivity extends AppCompatActivity {
-
   private static final String TAG = "FeedbackActivity";
-  private static final int THUMBNAIL_WIDTH = 200;
-  private static final int THUMBNAIL_HEIGHT = 200;
+  private static final int THUMBNAIL_WIDTH = 600;
+  private static final int THUMBNAIL_HEIGHT = -1; // scale proportionally
```

Comment:
I don't understand this, even with the comment.

Reply:
It means that the height depends on the width and is adjusted proportionally when scaling. E.g. the original image was 1800px wide and 3000px high. So with a target width of 600px and the proportional target height is determined to be 1000px to keep the image in proportion. A negative target means to keep the proportion. Any advice on how to explain that concisely?